test/test_grid.py

```diff
@@ -266,15 +266,16 @@ def test_face_areas_calculate_total_face_area_triangle():
     # validate the grid
     assert grid_verts.validate()
 
```

Comment:
Do we have any references face areas that we can compare our implementation too? Could be worth adding a test if we do.

Reply:
No, we don't - I'll look for one.